insights/combiners/package_provides_httpd.py

```diff
@@ -0,0 +1,65 @@
+"""
+PackageProvidesHttpdAll - Combiner for packages which provide httpd
+===================================================================
+
+Combiner for collecting all the httpd command and the corresponding package name
+which is parsed by the PackageProvidesHttpd parser.
+
+"""
+
+from insights.core.plugins import combiner
+from insights.parsers.package_provides_httpd import PackageProvidesHttpd
+from .. import LegacyItemAccess
+
+
+@combiner(PackageProvidesHttpd)
+class PackageProvidesHttpdAll(LegacyItemAccess):
+    """
+    Combiner for collecting all the httpd command and the corresponding package
+    name which is parsed by the PackageProvidesHttpd parser.
+    It works as a ``dict`` with the httpd command as the key and the
+    corresponding package name as the value.
+
+    Examples:
+        >>> from insights.tests import context_wrap
+        >>> PACKAGE_COMMAND_MATCH_1 = '''/usr/sbin/httpd httpd-2.4.6-88.el7.x86_64'''
+        >>> PACKAGE_COMMAND_MATCH_2 = '''/opt/rh/httpd24/root/usr/sbin/httpd httpd24-httpd-2.4.34-7.el7.x86_64'''
+        >>> pack1 = PackageProvidesHttpd(context_wrap(PACKAGE_COMMAND_MATCH_1))
+        >>> pack2 = PackageProvidesHttpd(context_wrap(PACKAGE_COMMAND_MATCH_2))
+        >>> packages = PackageProvidesHttpdAll([pack1, pack2])
```

Comment:
The lines above in the examples are not really how a developer would use this combiner, it would be less confusing if you just incorporate this into your test and then show the object type of `packages` and the remaining lines would be OK.  It would be helpful to add a bit more text above the examples that explains the combiner will receive a list of parsers, one for each running instance of httpd and each parser instance will contain the command information and the RPM package information for the respective `httpd` instance.